_posts/2023-02-01-solidity-0.8.18-release-announcement.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.18 Release Announcement'
+date: '2023-02-01'
+author: Solidity Team
+category: Releases
+---
+
+## Introducing the newest version of the Solidity Compiler!
+We are excited to announce the latest release of the Solidity Compiler, [Solidity v0.8.18](https://github.com/ethereum/solidity/releases/tag/v0.8.18). 
+This latest version includes a range of improvements and it also introduces support for the [Paris upgrade](https://blog.ethereum.org/2022/08/24/mainnet-merge-announcement)!
+
+
+### Notable New Features
+
+#### --no-cbor-metadataCommand-Line Flag
+ A new command-line flag (`--no-cbor-metadata`) and Standard JSON option (`settings.metadata.appendCBOR: false`) to prevent compiler from appending the CBOR metadata section at the end of the bytecode. Until now, it was only possible to exclude the hash of the JSON metadata file (`--metadata-hash none`/`settings.metadata.bytecodehash: "none"`) from that section, but the bytecode would still include a small bit of CBOR data containing the version of the compiler.
```

Comment:
```suggestion
#### --no-cbor-metadata Command-Line Flag
 A new command-line flag (`--no-cbor-metadata`) and Standard JSON option (`settings.metadata.appendCBOR: false`) to prevent compiler from appending the CBOR metadata section at the end of the bytecode. 
 Until now, it was only possible to exclude the hash of the JSON metadata file (`--metadata-hash none`/`settings.metadata.bytecodehash: "none"`) from that section, but the bytecode would still include a small bit of CBOR data containing the version of the compiler.
```